Components/ListSearchResults/Partials/Pagination/_style.scss

```diff
@@ -36,4 +12,12 @@
       }
     }
   }
+
+  &-link {
+    &[disabled] {
+      cursor: default;
+      opacity: 0.65;
+      pointer-events: none;
```

Comment:
I would prefer not showing buttons when they're not needed. You will probably need the floating approach then again.